src/MessagePack/MessagePackReader.Integers.cs

```diff
@@ -21,6 +22,7 @@ namespace MessagePack
         /// </summary>
         /// <returns>The value.</returns>
         /// <exception cref="OverflowException">Thrown when the value exceeds what can be stored in the returned type.</exception>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
You've only modified the `.cs` file here, which is T4 generated (as per the comment near the top of the file). You'd have to change the `.tt` file instead and let that update the `.cs` files.

Adding this attribute selectively to just certain types will be tricky, since the same pattern is applied to _all_ integer types.
I'm also wondering whether this is really a good method to inline anyway. It is rather large, so it may bloat your code size. Also, *why* is the code you're calling invoking this particular method repeatedly? If you have a lot of bytes to read, why not read them as a byte array instead, which is _much_ faster and more compact in the msgpack binary stream?